src/txn/txn_rollback_to_stable.c

```diff
@@ -1234,13 +1234,16 @@ __rollback_abort_fast_truncate(
  *     Skip if rollback to stable doesn't requires to read this page.
  */
 int
-__wt_rts_page_skip(WT_SESSION_IMPL *session, WT_REF *ref, void *context, bool *skipp)
+__wt_rts_page_skip(
+  WT_SESSION_IMPL *session, WT_REF *ref, void *context, bool visible_all, bool *skipp)
 {
     wt_timestamp_t rollback_timestamp;
 
     rollback_timestamp = *(wt_timestamp_t *)(context);
     *skipp = false; /* Default to reading */
 
+    WT_UNUSED(visible_all);
+
```

Comment:
This should go at the top of the function?

Reply:
I will hit an error if I do that.

Reply:
OK, fbm.